index.md

```diff
@@ -1,4 +1,127 @@
+        <i class="glyphicon glyphicon-globe"></i>
+        <section>
+          <h2>Runs in all browsers and implements .NET Standard</h2>
+          <p class="lead">Blazor runs in all browsers on the real .NET runtime with full support for [.NET Standard](https://docs.microsoft.com/dotnet/standard/net-standard). Blazor requires no plugins and no code transpilation, only open web standards.</p>
+          <a href="/docs/introduction/index.html" class="btn btn-primary">How Blazor works</a>
+        </section>
+      </div>
+    </div>
+</div>
+<div class="counter-key-section">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2">
+        <section>
+          <h2>Native performance with WebAssembly</h2>
+          <p class="lead">Runs on WebAssembly, giving you native performance in the browser and a trusted security sandbox.</p>
+        </section>
+        <i class="glyphicon glyphicon-fire"></i>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="key-section">
+  <div class="container content">
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2">
+        <i class="glyphicon glyphicon-transfer"></i>
+        <section>
+          <h2>Native browser apps</h2>
+          <p class="lead">Easily interact with your existing JavaScript code, your favorite libraries, and any browser API through JavaScript interop.</p>
+        </section>
+      </div>
+    </div>
+</div>
+<div class="counter-key-section">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2">
+        <section>
+          <h2>Simple and productive</h2>
+          <p class="lead">Get started fast and remain productive with project templates, great tooling, and reusable components.</p>
+          <a href="https://marketplace.visualstudio.com/items?itemName=aspnet.blazor" class="btn btn-primary">Tools for Blazor</a>
+        </section>
+        <i class="glyphicon glyphicon-console"></i>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="key-section">
+  <div class="container content">
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2">
+        <i class="glyphicon glyphicon-user"></i>
+        <section>
+          <h2>Get involved</h2>
+          <p class="lead">Join the community that's building Blazor, writing documentation, building samples, and more!</p>
+        </section>
+      </div>
+    </div>
+</div>
+<div class="counter-key-section">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2">
+        <section>
+          <h2>Open-source & free </h2>
+          <p class="lead">Blazor and the .NET platform are open-source with a strong community of 25,000+ contributors from over 1,700 companies.</p>
+          <a href="https://github.com/aspnet/blazor" class="btn btn-primary">Blazor on GitHub</a>
+        </section>
+        <i class="glyphicon glyphicon-road"></i>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
Can we have another Get Started button at the bottom of the page, similar to what they do on http://reactjs.org?

Reply:
Adding